src/shared/styles/theme.css

```
@@ -72,6 +110,12 @@
 
   --typo-caption2-size: 11px;
   --typo-caption2-line: 14px;
+
+
+  /* shadow */
+  --shadow-normal: 0px 0px 1px 0px rgba(0, 0, 0, 0.06), 0px 1px 4px 0px rgba(0, 0, 0, 0.08);
```

Stylelint (17.2.0)

[error] Expected "0.06" to be "6%" (alpha-value-notation)

(alpha-value-notation)